src/modules/ekf2/ekf2_params.c

```diff
@@ -1129,6 +1129,7 @@ PARAM_DEFINE_INT32(EKF2_RNG_AID, 0);
  * @group EKF2
  * @min 0.1
  * @max 2
+ * @unit m
```

Comment:
This should be m/s

Reply:
Thanks. Of course it should be :-(. Fixed.